packages/sources/src/components/pagination/pagination.args.ts

```diff
@@ -29,7 +29,10 @@ export function paginationArgsMapper(a: PaginationArgs): Pagination {
   return {
     totalPages: a.totalPages,
     currentPage: a.currentPage,
-    onSelectPage: a.onSelectPage,
+    onSelectPage: (event) => {
+      event.detail.originalEvent.preventDefault();
```

Comment:
@tfrijsewijk wat vind je hiervan? Ik twijfelde een beetje. Door dit te doen blijf je wel op je storybook pagina en zie je het event ook langskomen in de 'Actions'. Anders zul je dat tabje 'Actions' nooit echt kunnen zien werken.

Reply:
Leuk idee!